### ISSUE TITLE
Scaling factor for Gaussian precision and weight

### ISSUE THREAD

**Issue by Lewis-lv0** [leonidk/fmb-plus#5]
Hey @leonidk , thanks for this amazing work! Below is a code block in `run_co3d_sp.ipynb`, which is used to extract normals etc.  Could you please explain why do we need to apply the `scaleE` scaling factor for both `final_prec` and `final_weight_log`? I am also wondering why it is applied in different ways for `final_prec` and `final_weight_log`.  
```
points_export = []
colors_export = []
colors_export_plain = []

normals_export = []

scaleE = 2

thesh_min = 0.9
for i in range(len(poses)):
    pixel_list[:,3] = i
    rot_mats = jax.vmap(fm_render.quat_to_rot)(poses[:,:4])
    def rot_ray_t(rayi):
        ray = rayi[:3] * jnp.array([invF,invF,1])
        pose_idx = rayi[3].astype(int)
        return jnp.array([ray@rot_mats[pose_idx],poses[pose_idx][4:]])
    camera_rays_start = jax.vmap(rot_ray_t)(pixel_list)
    est_depth,est_alpha,est_norm,est_w,flowp,flowm = render_jit2(final_mean, final_prec*scaleE,(scaleE**2)*final_weight_log,pixel_list,invF,poses)
```

**Comment by leonidk** [comment]
That's a great question! 

The motivation behind that was to create smaller, denser Gaussians and allow the Poisson Reconstruction to do more interpolation. In practice, I later found that using `thresh_min` with `render_jit2` ended up as a better way to achieve the same behavior.

The reason it's applied differently is because the precisions are the inverse square root in scale and the weights are linear in scale. So to make the Gaussians denser, the idea was to increase the weight, and scale the covariance down. But `final_prec` is the inverse square root of covariance, hence the squared difference in the terms. 

TLDR: `scaleE` is legacy code of me testing some idea that's not very important and can be ignored.
